src/languages/en_US.json

```diff
@@ -1978,6 +1978,7 @@
         "swapping": "Swapping",
         "select_token": "Select Token",
         "insufficient_funds": "Insufficient Funds",
+        "insufficient_gas": "Insufficient Gas",
```

Comment:
I'm not sure this label is clear to the user, however I don't have a better alternative off the top of my head. To me this reads as though we're failing to estimate gas or something. Insufficient funds for gas is better but too wordy. Will need to think on this.

Reply:
another option I can think is `Insufficient <native>` like `Insufficient ETH`, `Insufficient BNB`

Reply:
insufficient <native> i think is better tbh and more clear on what the issue is